### ISSUE TITLE
Support for mixed=True

### ISSUE THREAD

**Issue by AmatanHead** [brunato/xmlschema#46]
Hello!

I'm trying to parse a compound type with mixed contents. An example of such type can be found here: https://www.w3schools.com/xml/schema_complex_mixed.asp

The result is unsatisfactory:

```python
{'name': 'John Smith', 'orderid': 1032, 'shipdate': '2001-07-13'}
```

I can handle this issue by writing my own `XMLSchemaConverter`, however, it would be very nice to have a builtin way to work with such elements.

**Comment by brunato** [comment]
It's already provided by the `XMLSchemaConverter` class, but It doesn't work for a bug at line 144 of `converters.py`:

https://github.com/brunato/xmlschema/blob/0d4e2188aa0e51db93a9de2d49532e39d18dac0f/xmlschema/converters.py#L144

`xsd_child is None` for a CDATA section, so this line generate an exception and the section is not decoded.
I've fixed this into my development. I'll merge the fix into the next release.

Best regards
Davide

**Comment by brunato** [comment]
A fix is available with version 0.9.23.
I close this issue, reopen if this doesn't satisfy your question.
